dask/array/core.py

```diff
@@ -2831,6 +2831,21 @@ def from_array(
     >>> import dask.base
     >>> token = dask.base.tokenize(x)  # doctest: +SKIP
     >>> a = da.from_array('myarray-' + token)  # doctest: +SKIP
+
+    Numpy ndarrays that are smaller than array.chunk-size are wholly embedded in the graph,
+    communicated to all workers and then sliced at compute time. So operator.getitem
+    tasks are present in the graph.
+    >>> import dask.array as da
```

Comment:
These might need endlines to render well?  If you haven't already can I ask you to try making the docs and taking a look?

Something like this might work

```
cd docs
pip install -r requirements-docs.txt
make html
open build/html/index.html
```

Or alternatively we could just put in endlines regardless

Reply:
done :+1:
![image](https://user-images.githubusercontent.com/2448579/92519496-8f3d2d00-f209-11ea-8c52-77419569cb6f.png)